### PR TITLE
E2-2013: add api calls to request queue

### PR DESCRIPTION
**Pull request by e7mac**
What              | Where/Who
------------------|----------------------------------------
JIRA Issue        | [E2-2013](https://leanplum.atlassian.net/browse/E2-2013)
People Involved   | @hrishileanplum , @gg4race 

## Background
We need the API calls to be added to a queue, not called instantly.

## Implementation
As discussed

## Testing steps
Unit tests.

### CODE REVIEW

File: Leanplum/Api/LPUserApi.m [hunk 1]
```diff
@@ -49,11 +50,20 @@ + (void) setUserId:(NSString *)userId
     if (attributes != nil) {
         params[LP_PARAM_USER_ATTRIBUTES] =  [LPJSON stringFromJSON:attributes];
     }
-    LPWSManager *wsManager = [[LPWSManager alloc] init];
-    [wsManager sendPOSTWebService:[LPApiMethods getApiMethod:LPApiMethodSetUserAttributes]
-                       withParams:params
-                     successBlock:successResponse
-                     failureBlock:failureResponse];
+
+//    if ([LPApiConstants sharedState].isMulti) {
```
Comment (hrishileanplum) [inline Leanplum/Api/LPUserApi.m]: Not sure this should be commented out. Also missing the logic for responseDictionaryFromResponse

File: LeanplumTests/LeanplumTests.m [hunk 2]
```diff
@@ -35,43 +35,43 @@ - (void)testConfiguration
 {
     NSString* host = @"test_host";
     NSString* servlet = @"servlet";
-
```
Comment (hrishileanplum) [inline LeanplumTests/LeanplumTests.m]: Ist this just text formatting for this file?

File: Leanplum/Api/LPUserApi.m (hunk 1 above)
Comment (e7mac) [reply Leanplum/Api/LPUserApi.m]: Missing the logic? What do you mean? It's there on L27, and the method is in `LPAPIUtils`

File: LeanplumTests/LeanplumTests.m (hunk 2 above)
Comment (e7mac) [reply LeanplumTests/LeanplumTests.m]: i think these parts it formatted the text, but the file has other changes as well

**Pull request merged**